quipucords/tests/fingerprinter/test_fingerprint_task.py

```diff
@@ -1531,9 +1531,9 @@ def test_process_details_report_success(fingerprint_task_runner):
         "ip_addresses": ["1.2.3.4"],
         "sources": [],
     }
-    deployments_report = DeploymentsReport(report_id=1, id=1)
+    deployments_report = DeploymentsReport(id=1)
     deployments_report.save()
-    details_report = DetailsReport(id=1, deployment_report=deployments_report)
+    details_report = Report(id=1, deployment_report=deployments_report)
```

Comment:
I look forward to all these hard-coded `1`s disappearing in the near future. 🙂 

(Not suggesting you need to change these now. Just noticing how quickly your _other_ recent change will become relevant and useful.)